### PR TITLE
doc for sdk release

### PR DESCRIPTION
**Pull request by rohithkumar91**

### CODE REVIEW

File: sdk/azurestackhci/Azure.ResourceManager.Hci/tests/SessionRecords/HciClusterOperationTests/GetUpdateDelete().json [hunk 1]
```diff
@@ -1,46 +1,46 @@
...
-        "managedByTenants": [],
-        "subscriptionId": "a9280d5f-f9ab-436e-90cb-b6b9863c5551",
-        "tenantId": "72f988bf-86f1-41af-91ab-2d7cd011db47",
-        "displayName": "ASME_RestonAzureStackEdge_Test",
-        "state": "Enabled",
-        "subscriptionPolicies": {
-          "locationPlacementId": "Internal_2014-09-01",
-          "quotaId": "Internal_2014-09-01",
-          "spendingLimit": "Off"
+        "RequestUri": "https://management.azure.com/subscriptions/a9280d5f-f9ab-436e-90cb-b6b9863c5551?api-version=2023-02-01",
```
Comment (live1206) [inline sdk/azurestackhci/Azure.ResourceManager.Hci/tests/SessionRecords/HciClusterOperationTests/GetUpdateDelete().json:4]:
```suggestion
        "RequestUri": "https://management.azure.com/subscriptions/a9280d5f-f9ab-436e-90cb-b6b9863c5551?api-version=2021-01-01",
```

File: sdk/azurestackhci/Azure.ResourceManager.Hci/tests/SessionRecords/HciClusterOperationTests/GetUpdateDelete()Async.json [hunk 2]
```diff
@@ -1,46 +1,46 @@
...
-        "managedByTenants": [],
-        "subscriptionId": "a9280d5f-f9ab-436e-90cb-b6b9863c5551",
-        "tenantId": "72f988bf-86f1-41af-91ab-2d7cd011db47",
-        "displayName": "ASME_RestonAzureStackEdge_Test",
-        "state": "Enabled",
-        "subscriptionPolicies": {
-          "locationPlacementId": "Internal_2014-09-01",
-          "quotaId": "Internal_2014-09-01",
-          "spendingLimit": "Off"
+        "RequestUri": "https://management.azure.com/subscriptions/a9280d5f-f9ab-436e-90cb-b6b9863c5551?api-version=2023-02-01",
```
Comment (live1206) [inline sdk/azurestackhci/Azure.ResourceManager.Hci/tests/SessionRecords/HciClusterOperationTests/GetUpdateDelete()Async.json:4]:
```suggestion
        "RequestUri": "https://management.azure.com/subscriptions/a9280d5f-f9ab-436e-90cb-b6b9863c5551?api-version=2021-01-01",
```

File: sdk/azurestackhci/Azure.ResourceManager.Hci/tests/SessionRecords/ArcSettingCollectionTests/CreateGetList().json [hunk 3]
```diff
@@ -1,46 +1,46 @@
...
-        "managedByTenants": [],
-        "subscriptionId": "a9280d5f-f9ab-436e-90cb-b6b9863c5551",
-        "tenantId": "72f988bf-86f1-41af-91ab-2d7cd011db47",
-        "displayName": "ASME_RestonAzureStackEdge_Test",
-        "state": "Enabled",
-        "subscriptionPolicies": {
-          "locationPlacementId": "Internal_2014-09-01",
-          "quotaId": "Internal_2014-09-01",
-          "spendingLimit": "Off"
+        "RequestUri": "https://management.azure.com/subscriptions/a9280d5f-f9ab-436e-90cb-b6b9863c5551?api-version=2023-02-01",
```
Comment (live1206) [inline sdk/azurestackhci/Azure.ResourceManager.Hci/tests/SessionRecords/ArcSettingCollectionTests/CreateGetList().json:4]:
```suggestion
        "RequestUri": "https://management.azure.com/subscriptions/a9280d5f-f9ab-436e-90cb-b6b9863c5551?api-version=2021-01-01",
```

File: sdk/azurestackhci/Azure.ResourceManager.Hci/tests/SessionRecords/ArcSettingCollectionTests/CreateGetList()Async.json [hunk 4]
```diff
@@ -1,46 +1,46 @@
...
-        "managedByTenants": [],
-        "subscriptionId": "a9280d5f-f9ab-436e-90cb-b6b9863c5551",
-        "tenantId": "72f988bf-86f1-41af-91ab-2d7cd011db47",
-        "displayName": "ASME_RestonAzureStackEdge_Test",
-        "state": "Enabled",
-        "subscriptionPolicies": {
-          "locationPlacementId": "Internal_2014-09-01",
-          "quotaId": "Internal_2014-09-01",
-          "spendingLimit": "Off"
+        "RequestUri": "https://management.azure.com/subscriptions/a9280d5f-f9ab-436e-90cb-b6b9863c5551?api-version=2023-02-01",
```
Comment (live1206) [inline sdk/azurestackhci/Azure.ResourceManager.Hci/tests/SessionRecords/ArcSettingCollectionTests/CreateGetList()Async.json:4]:
```suggestion
        "RequestUri": "https://management.azure.com/subscriptions/a9280d5f-f9ab-436e-90cb-b6b9863c5551?api-version=2021-01-01",
```

File: sdk/azurestackhci/Azure.ResourceManager.Hci/tests/SessionRecords/HciClusterCollectionTests/CreateGetList().json [hunk 5]
```diff
@@ -1,46 +1,46 @@
...
-        "managedByTenants": [],
-        "subscriptionId": "a9280d5f-f9ab-436e-90cb-b6b9863c5551",
-        "tenantId": "72f988bf-86f1-41af-91ab-2d7cd011db47",
-        "displayName": "ASME_RestonAzureStackEdge_Test",
-        "state": "Enabled",
-        "subscriptionPolicies": {
-          "locationPlacementId": "Internal_2014-09-01",
-          "quotaId": "Internal_2014-09-01",
-          "spendingLimit": "Off"
+        "RequestUri": "https://management.azure.com/subscriptions/a9280d5f-f9ab-436e-90cb-b6b9863c5551?api-version=2023-02-01",
```
Comment (live1206) [inline sdk/azurestackhci/Azure.ResourceManager.Hci/tests/SessionRecords/HciClusterCollectionTests/CreateGetList().json:4]:
```suggestion
        "RequestUri": "https://management.azure.com/subscriptions/a9280d5f-f9ab-436e-90cb-b6b9863c5551?api-version=2021-01-01",
```

File: sdk/azurestackhci/Azure.ResourceManager.Hci/tests/SessionRecords/HciClusterCollectionTests/CreateGetList()Async.json [hunk 6]
```diff
@@ -1,46 +1,46 @@
...
-        "managedByTenants": [],
-        "subscriptionId": "a9280d5f-f9ab-436e-90cb-b6b9863c5551",
-        "tenantId": "72f988bf-86f1-41af-91ab-2d7cd011db47",
-        "displayName": "ASME_RestonAzureStackEdge_Test",
-        "state": "Enabled",
-        "subscriptionPolicies": {
-          "locationPlacementId": "Internal_2014-09-01",
-          "quotaId": "Internal_2014-09-01",
-          "spendingLimit": "Off"
+        "RequestUri": "https://management.azure.com/subscriptions/a9280d5f-f9ab-436e-90cb-b6b9863c5551?api-version=2023-02-01",
```
Comment (live1206) [inline sdk/azurestackhci/Azure.ResourceManager.Hci/tests/SessionRecords/HciClusterCollectionTests/CreateGetList()Async.json:4]:
```suggestion
        "RequestUri": "https://management.azure.com/subscriptions/a9280d5f-f9ab-436e-90cb-b6b9863c5551?api-version=2021-01-01",
```

File: sdk/azurestackhci/Azure.ResourceManager.Hci/src/autorest.md [hunk 7]
```diff
@@ -9,7 +9,7 @@ generate-model-factory: false
 csharp: true
 library-name: Hci
 namespace: Azure.ResourceManager.Hci
-require: https://github.com/Azure/azure-rest-api-specs/blob/7d1f6b268def6736833a08311c87cc96740eaf03/specification/azurestackhci/resource-manager/readme.md
+require: https://github.com/Azure/azure-rest-api-specs/blob/51b8f2084b05e705baf30f24ea0a40a346e2c5a5/specification/azurestackhci/resource-manager/readme.md
```
Comment (live1206) [inline sdk/azurestackhci/Azure.ResourceManager.Hci/src/autorest.md:12]: As I can see, the latest readme should be https://github.com/Azure/azure-rest-api-specs/blob/3eaf771dc5323403a3144a8f731083b7d9ea4f97/specification/azurestackhci/resource-manager/readme.md

Merge the latest main branch to your branch, and try to regenerate code.

**Comment by rohithkumar91** [comment]
@microsoft-github-policy-service agree company="Microsoft"

File: sdk/azurestackhci/Azure.ResourceManager.Hci/src/autorest.md (hunk 7 above)
Comment (rohithkumar91) [reply sdk/azurestackhci/Azure.ResourceManager.Hci/src/autorest.md]: Done

**Review by live1206 — APPROVED** [review]

**Pull request merged**